### ISSUE TITLE
breaks on node 19: Error [ERR_MODULE_NOT_FOUND]: Cannot find module

### ISSUE THREAD

**Issue by milahu** [Shakeskeyboarde/slashes#7]
im trying to use `slashes` in an ESM package

first node says

```
import { addSlashes, removeSlashes } from 'slashes'
         ^^^^^^^^^^
SyntaxError: Named export 'addSlashes' not found. The requested module 'slashes' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'slashes';
const { addSlashes, removeSlashes } = pkg;
```

fix: add `"type": "module",` to node_modules/slashes/package.json

then node says

> Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'node_modules/slashes/lib/module/add-slashes' imported from node_modules/slashes/lib/module/index.js

fix: add `.js` extension to imports

<details>
<summary>Details</summary>

```diff
diff --git a/node_modules/slashes/lib/module/add-slashes.js b/node_modules/slashes/lib/module/add-slashes.js
index 14f33f5..a70f2f4 100644
--- a/node_modules/slashes/lib/module/add-slashes.js
+++ b/node_modules/slashes/lib/module/add-slashes.js
@@ -1,5 +1,5 @@
-import { getEscapedAny } from './get-escaped-any';
-import { getEscapedJsonUnsafe } from './get-escaped-json-unsafe';
+import { getEscapedAny } from './get-escaped-any.js';
+import { getEscapedJsonUnsafe } from './get-escaped-json-unsafe.js';
 const addSlashes = (str, { getEscaped = getEscapedJsonUnsafe } = {}) => {
     let result = '';
     for (const char of str) {
diff --git a/node_modules/slashes/lib/module/index.js b/node_modules/slashes/lib/module/index.js
index 5c6d8bb..132076b 100644
--- a/node_modules/slashes/lib/module/index.js
+++ b/node_modules/slashes/lib/module/index.js
@@ -1,6 +1,6 @@
-export * from './add-slashes';
-export * from './get-escaped-any';
-export * from './get-escaped-json-unsafe';
-export * from './get-unescaped-any';
-export * from './remove-slashes';
+export * from './add-slashes.js';
+export * from './get-escaped-any.js';
+export * from './get-escaped-json-unsafe.js';
+export * from './get-unescaped-any.js';
+export * from './remove-slashes.js';
 //# sourceMappingURL=index.js.map
diff --git a/node_modules/slashes/lib/module/remove-slashes.js b/node_modules/slashes/lib/module/remove-slashes.js
index 8b65849..3f60ef9 100644
--- a/node_modules/slashes/lib/module/remove-slashes.js
+++ b/node_modules/slashes/lib/module/remove-slashes.js
@@ -1,4 +1,4 @@
-import { getUnescapedAny } from './get-unescaped-any';
+import { getUnescapedAny } from './get-unescaped-any.js';
 const removeSlashes = (source, { getUnescaped = getUnescapedAny } = {}) => {
     const rx = /(?:(\\(u([0-9a-f]{4})|u\{([0-9a-f]+)\}|x([0-9a-f]{2})|(\d{1,3})|([\s\S]|$)))|([\s\S]))/giu;
     let match;
```

</details>

to autofix the typescript imports see
https://github.com/milahu/random/blob/master/javascript/typescript-autofix-js-import-extension.js

**Comment by Shakeskeyboarde** [comment]
Fixed in PR #8 (v3.0.11). Let me know if that doesn't fix it.

**Comment by milahu** [comment]
works, thanks : )
